package.json

```diff
@@ -31,6 +30,16 @@
   },
   "devDependencies": {
     "enzyme": "^3.10.0",
-    "enzyme-adapter-react-16": "^1.14.0"
+    "enzyme-adapter-react-16": "^1.14.0",
+    "puppeteer": "^1.19.0"
+  },
+  "jest": {
+    "collectCoverageFrom": [
+      "src/**/*.{js,jsx}",
```

Comment:
So this config tells Jest to ignore on coverage calculation these files?

Reply:
yes